crates/router/src/connector/payme/transformers.rs

```diff
@@ -879,9 +885,9 @@ impl<F, T>
         item: types::ResponseRouterData<F, CaptureBuyerResponse, T, types::PaymentsResponseData>,
     ) -> Result<Self, Self::Error> {
         Ok(Self {
-            payment_method_token: Some(types::PaymentMethodToken::Token(
+            payment_method_token: Some(types::PaymentMethodToken::Token(Secret::new(
                 item.response.buyer_key.clone().expose(),
-            )),
+            ))),
```

Comment:
You can probably get rid of the `Secret::new()` and the `expose()`?

```rust
payment_method_token: Some(types::PaymentMethodToken::Token(item.response.buyer_key.clone())),
```

Reply:
Done